src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/mathshare?user=postgres&password=mysecretpassword&ssl=true"
```

Comment:
why ssl true? for dev, most people won't have ssl enabled I imagine

Reply:
I've followed the suggestions from here: https://jdbc.postgresql.org/documentation/head/connect.html, and additionally, it wasn't working for me with disabled SSL (with default PostgreSQL configuration), so I left it enabled.

Reply:
My mistake, there is an additional character ```"``` at the end of this line, and in this case, the app treats the value as false. I'll remove it as it's indeed not necessary.